### PR TITLE
Update demos.yaml

### PR DESCRIPTION
**Pull request by LightGuard**
Demo locations changed....

### CODE REVIEW

**Comment by LightGuard** [comment]
This may no longer be needed.

File: demos.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - https://raw.githubusercontent.com/jboss-developer/jboss-developer-demos/master/fusebyexample/external-mq-fabric-client.yaml 
 - https://raw.githubusercontent.com/jboss-developer/jboss-developer-demos/master/fusebyexample/kie-remote-example.yaml
-- https://raw.githubusercontent.com/weimeilin79/iot-mashup/master/iot-mashup.yaml
```
Comment (paulrobinson) [inline demos.yaml]: This removal is no longer required.

**Comment by paulrobinson** [comment]
Needs rebasing. Then can be merged once the following is merged: https://github.com/jbossdemocentral/jboss-fuse-iot-mashup/pull/1

**Comment by paulrobinson** [comment]
Closed as it's probably our of date. Please re-open (and rebase) if you think otherwise.

**Pull request closed without merging**